mappings/net/minecraft/client/network/ClientboundLoginNetworkHandler.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS dci net/minecraft/client/network/ClientboundLoginNetworkHandler
```

Comment:
I dunno if Clientbound fits here. Same with ClientPlayNetworkHandler. Those are both client-*side*, instead of client-*bound*.

Reply:
Same goes for `ServerLoginNetworkHandler` and friends. Probably a find-and-replace error.